ui/perfherder/helpers.js

```diff
@@ -21,6 +21,12 @@ import {
   phTimeRanges,
 } from './constants';
 
+export const formatNumber = input => {
+  return new Intl.NumberFormat('en-US', { maximumFractionDigits: 2 }).format(
```

Comment:
Since it's only one line of code (even though its formatted to be multi-line) you can remove the curly brackets and the return statement like so: 
```
export const formatNumber = input => new Intl.NumberFormat('en-US', { maximumFractionDigits: 2 }).format(
    input,
  );
```